joomla-projects/GSoC17_publishing_workflow
# Fix category issue

Pull Request for Issue 49 .

### Summary of Changes
Remove 'workflow_id' field from #__categories table structure


### Testing Instructions
Goto category view and create a new category, you will get  'workflow_id' does not have a default value. After reinstalling with new schema, you won't see that error and category is saving successfully.


### Expected result
Before fixing:  'workflow_id' does not have a default value.


### Actual result
After fixing: saving the category successfully


### Documentation Changes Required
No
